db/migrate/20230804151059_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :surname, :string
+  end
```

Comment:
| Error in Browser | Error in Terminal |
|-|-|
| ![image](https://github.com/massazoum/Rails_App/assets/93039359/0d0bc9bb-0438-4110-8cd7-8a030973c25f) | ![image](https://github.com/massazoum/Rails_App/assets/93039359/ec0ec058-7b1b-448a-8570-dbb309f8f907) |

- [ ] As suggested by the previous reviewer that your migrations are giving some error kindly fix these migration files to make your migrations done successfully because if any of the migrations get stuck then your app won't run :fearful: KIndly fix it :+1:

Quick Fix: I have fixed this error by removing line 3 which is 👇

```ruby
  add_column :users, :name, :string
```